tests/suites/test_suite_ssl.data

```diff
@@ -276,7 +276,7 @@ depends_on:MBEDTLS_CCM_C:MBEDTLS_AES_C:MBEDTLS_RSA_C:MBEDTLS_ECP_DP_SECP384R1_EN
 handshake_cipher:"TLS-RSA-WITH-AES-128-CCM":MBEDTLS_PK_RSA:0
 
 Handshake, DHE-RSA-WITH-AES-256-CBC-SHA256
-depends_on:MBEDTLS_CIPHER_MODE_CBC:MBEDTLS_AES_C:MBEDTLS_HAS_ALG_SHA_256_VIA_MD_OR_PSA_BASED_ON_USE_PSA:MBEDTLS_RSA_C:MBEDTLS_ECP_DP_SECP384R1_ENABLED:MBEDTLS_KEY_EXCHANGE_DHE_RSA_ENABLED
+depends_on:MBEDTLS_CIPHER_MODE_CBC:MBEDTLS_AES_C:MBEDTLS_HAS_ALG_SHA_256_VIA_MD_OR_PSA_BASED_ON_USE_PSA:MBEDTLS_RSA_C:MBEDTLS_KEY_EXCHANGE_DHE_RSA_ENABLED:MBEDTLS_DHM_C
```

Comment:
We don't actually need a dependency on `MBEDTLS_DHM_C` here (and all the other changed lines in this file), since that's implied by `MBEDTLS_KEY_EXCHANGE_DHE_RSA_ENABLED`. For the same reason, these lines don't declare a dependency on RSA either.

Please don't add a dependency on `MBEDTLS_DHM_C`, which is only needed when `MBEDTLS_USE_PSA_CRYPTO` is enabled.